src/t2iapi/device/service.proto

```diff
@@ -27,6 +27,8 @@ service DeviceService {
     /*
     Switch the device into a specific operating mode.
     This manipulation shall result in a Mds having a specific MdsOperatingMode.
+    The manipulated state shall be persistent until a next manipulation call or until the end of the test case
+    execution.
      */
```

Comment:
what do you think about adding:  "If the device is not able to maintain the static state, it shall return not supported."

Reply:
yes, I would add this sentence too

Reply:
added